canon-docs/scripts/build-css.js

```diff
@@ -13,7 +13,7 @@ const bundleAndCopyFile = async (source, destination) => {
   try {
     const result = await bundle({
       filename: source,
-      minify: true,
```

Comment:
Is this an intended change?

Reply:
We did remove the minify in the package per @Rugvip's request but you're right in that case it's better to keep it minified and make it faster for production. I also think that Next.js is bundling this CSS so this would not have any effect.

Reply:
I think that we would minify it anyways in our tooling, was just checking.

Happy to do either. If we're going to keep the `minification` on though we should regenerate the styles right?